widelands-build19-ppc64le.patch

```diff
@@ -0,0 +1,12 @@
+diff -up widelands-build19-src-gcc7/CMakeLists.txt.ppc64le widelands-build19-src-gcc7/CMakeLists.txt
```

Comment:
We don't support ppc64le so I don't see the point, unless it fixes something for other arches.